dev-docs/modules/reconciliationRtdProvider.md

```diff
@@ -0,0 +1,82 @@
+---
+layout: page_v2
+title: Reconciliation SDK Module
+description: Reconciliation Real Time Data Module
+page_type: module
+module_type: rtd
+module_code: reconciliation
+display_name: Reconciliation
+enable_download: true
+sidebarType: 1
+---
+
+# Reconciliation SDK Module
+{:.no_toc}
+
+* TOC
+{:toc}
+
+## Overview
+
+The purpose of Reconciliation Real Time Data Provider is to allow publishers to collect supply chain structure information and vendor-specific impression IDs from suppliers participating in ad creative delivery and report it to the Reconciliation Service, allowing publishers, advertisers and other supply chain participants to match and reconcile ad server, SSP, DSP and veritifation system log file records. 
```

Comment:
Better, but this is still a hard-to-understand sentence and has a typo "veritifation"

Try this:
```
The purpose of Reconciliation Real Time Data Provider is to allow publishers to collect supply chain structure information and vendor-specific impression IDs from ad creative suppliers. It reports the schains to the Reconciliation Service, allowing publishers, advertisers and other supply chain participants to match and reconcile ad server, SSP, DSP and verification system log file records. 
```

How does someone get an account with Reconcilliation? Normally there's contact info. Scanning the press release, maybe contact info is http://tagtoday.net/?

Reply:
@bretg, thank you a lot for the proposal.
I've updated the description and added the contact info:
consultation@tagtoday.net